gmso/core/box.py

```diff
@@ -117,30 +117,75 @@ class Box(object):
 
     """
 
-    def __init__(self, lengths, angles=None):
+    def __init__(self, lengths, angles=None, precision=None):
         """Construct a `Box` based on lengths and angles."""
-        self._lengths = _validate_lengths(lengths)
-        self._angles = _validate_angles(angles)
+        print(lengths, angles)
+        lengths = _validate_lengths(lengths)
+        angles = _validate_angles(angles)
+        print(lengths, angles)
+        precision = int(precision or 6)
```

Comment:
Does this set precision to 6 if you cant typecast precision to an int?

Reply:
I think it does because you have handled that in Molbox.

Reply:
Ah true, that makes sense.